R/derive_bds_params.R

```diff
@@ -0,0 +1,250 @@
+#' Adds a Ratio Parameter Computed from the Analysis Value of Other Parameters
```

Comment:
If this function is a utility for the other two, in my opinion it should not have its own file - it should instead be at the top of the other file.

Reply:
Negotiable but I agree to move it.

Reply:
Done. Everything is in a single file now.